ui/src/app/components/object-detail/object-detail.component.html

```diff
@@ -225,7 +225,7 @@ <h3 class="title">
             <ng-container matColumnDef="spColName">
               <th mat-header-cell class="table_header" *matHeaderCellDef>Name</th>
               <td mat-cell *matCellDef="let element">
-                <div *ngIf="isEditMode && element.get('spOrder').value !== ''">
```

Comment:
Still getting undefined_undefined_undefined.json after saving session then resuming and then interleaving the table

Reply:
Updated